### ISSUE TITLE
The CLI gives unhelpful error messages if --notes are not given

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2214]
I've not identified the cause of the issue yet, but I noticed that the CLI cannot create updates:

```
$ bodhi updates new s2i-core-0-13.f27container --url http://localhost:6543
Required
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
```

**Comment by bowlofeggs** [comment]
Surprisingly, it seems that the error is due to missing notes. If I include a ```--notes``` flag, we do not get the above error:

```
$ bodhi updates new s2i-core-0-13.f27container --notes "a note"
Invalid tag: s2i-core-0-13.f27container not tagged with any of the following tags [u'f22-updates-candidate', u'dist-6E-epel-testing-candidate', u'f21-updates-candidate', u'f25-updates-candidate', u'f24-updates-candidate', u'epel7-testing-candidate', u'f27-modular-updates-candidate', u'dist-5E-epel-testing-candidate', u'f23-updates-candidate', u'f26-updates-candidate', u'f27-updates-candidate', u'f28-updates-candidate', u'f28-modular-updates-candidate', u'f27container-updates-candidate']
```

Thus it seems that we simply have an error message propagation issue, which is important to fix but not critical.

**Comment by bowlofeggs** [comment]
Note: This "smells" like an API issue to me, but @cverna pointed out that it would be good for the CLI to also require the notes field. Thus, I suggest ensuring that the API hands back a reasonable error message in this case (for non CLI users) and that the CLI just errors immediately when notes aren't given (for CLI users).

**Comment by bowlofeggs** [comment]
A fix for this issue is planned to be included in the upcoming 3.9.0 release: #2477.

**Comment by bowlofeggs** [comment]
3.9.0 was released.
